### DISCUSSION TITLE
How to download pytorch cpu in requirements.txt

### DISCUSSION THREAD

**Discussion by carolksun** [deta/cloud-docs#443] [Help]
I'm struggling to download pytorch cpu onto my deta micro. I know that pytorch itself is too big to download so I would like to download the cpu only version. My micro has 512 mb of memory and 10s for timeout (maybe this is the problem?).

For now, my `main.py` is the default "Hello World", so I know that it is not an issue with this. It works fine without the pytorch dependency.

My requirements.txt is just
```
fastapi
numpy
pytorch -- want to download but can't
```

I've tried using the link to the wheel in requirements.txt
`http://download.pytorch.org/whl/cpu/torch-0.4.1-cp36-cp36m-linux_x86_64.whl` which resulted in `Internal server error`.

I've also tried to add the flag of 
```
-f https://download.pytorch.org/whl/torch_stable.html
torch==1.11.0+cpu
```
but it says it's an `unsupported option`.

Has anyone been able to download the torch-cpu package successfully on their micro?

**Comment by abdelhai** [comment]
i don't believe this behavior is supported on deta micros yet. i'm taking a note to see if we could support it. cc @aavshr
